core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -68,6 +68,16 @@
 
         <div role="form" class="form-horizontal">
           <div class="form-group">
+            <label class="col-lg-2 col-md-2 col-sm-2">Preferred Dashboard</label>
+            <div class="col-lg-10 col-md-10 col-sm-10">
+              <input type="radio" ng-model="defaultDashboard" value="creator" ng-change="saveDefaultDashboard(defaultDashboard)"> Creator Dashboard<br>
```

Comment:
Can we wrap these in `<label>` tags (like the email checkboxes below) so the user can click on the words and not just the radio button?